hackclub.com.yaml

```diff
@@ -1687,6 +1687,21 @@ orpheushacks: # Harrison Katz automatically wins for adding the DNS entry - that
   - ttl: 1
     type: CNAME
     value: cname.vercel-dns.com.
+outerlan:
+  - ttl: 1
+    type: A
+    values:
+      - 185.199.108.153.
+      - 185.199.109.153.
+      - 185.199.110.153.
+      - 185.199.111.153.
```

Comment:
remove the dot after the IP addresses I think

Reply:
alright, thank you